### PR TITLE
Add support for `initialPage` option

### PR DESCRIPTION
**Pull request by geun**
Direct to signUp page with initialPage options
Lock has this options but it didn't have these options . 

Ref [lock/v11/configuration#initialscreen-string](https://auth0.com/docs/libraries/lock/v11/configuration#initialscreen-string-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,23 @@ app.get(
 );
 ```
 
+### initialPage
+
+If you want to change initial Page to sign up, set `mode` or `initialPage`(hosted):
+Values should be matched `login`, `signUp`, and `forgotPassword`
+
+[lock-v11-configuration](https://auth0.com/docs/libraries/lock/v11/configuration#initialscreen-string-)
```
Comment (luisrudge) [inline README.md]: I feel this needs a better explanation on how to make it work. Also, it'd be good to note that this only works when using Universal Login Page's in Classic mode. It won't work in the new ULP.

Comment (geun) [reply README.md]: @luisrudge We need to use new Universal login. Is there any plan to implements?

Comment (luisrudge) [reply README.md]: It's in the backlog, but the ULP team has other priorities right now.

Comment (geun) [reply README.md]: @luisrudge thanks for replying, I'll update it after release that.

**Comment by timowang1991** [comment]
@geun  @luisrudge  Hi, is there a reason to why we cannot have this feature available? Thanks!

**Pull request closed without merging**